crates/rspack_core/src/stats.rs

```diff
@@ -418,6 +421,13 @@ impl Stats<'_> {
       reasons,
       assets,
       modules,
+      source: source
+        .then(|| {
+          module
```

Comment:
You can keep this field and use `Option<&dyn Source>` instead of eagerly converting it to a `String`